specs/vep-2272-complete-data-job-configuration-persistence/deploy_data_job_diagram.puml

```diff
@@ -0,0 +1,31 @@
+@startuml
```

Comment:
I think you need to update the diagrams to reflect the change in the high level design.